src/main/features/ccj/views/check-and-send.njk

```diff
@@ -33,6 +33,10 @@
       draft.paidAmount.option.value | capitalize, changeLink = paidAmountUrl, bottomBorder = false) }}
     {{ row('Amount to be paid by defendant', amountToBePaid | numeral) }}
 
+    {% if draft.paidAmount.amount %}
```

Comment:
Have the content and position been agreed with UX / PO?

Why there is no change link next to the paid amount? Why isn't that in the same group as two previous lines?

<img width="988" alt="screen shot 2018-04-12 at 02 43 19" src="https://user-images.githubusercontent.com/1724691/38651884-68fb8ef8-3dfb-11e8-8d83-29ee9b12795a.png">

Reply:
The change links seems to be correct but the content and layout isn't
See the screenshot on:
https://tools.hmcts.net/jira/browse/ROC-3379